web/post-restoration-transition-application/app/components/Connect/Form/Section.vue

```diff
@@ -11,10 +11,16 @@ defineProps<{
     :class="[error ? 'border-l-2 border-red-600' : '']"
   >
     <div class="flex flex-col sm:flex-row">
-      <div v-if="title" class="w-[200px] lg:w-[320px]">
-        <p class="mb-2 sm:mb-0 sm:font-bold" :class="error ? 'text-red-600' : ''">
-          {{ title }}
-        </p>
+      <div
+        v-if="$slots.title || title"
+        class="w-[200px] lg:w-[320px]"
+        :class="error ? 'text-red-600' : ''"
```

Comment:
did you test that moving the error class here works? I am thinking the default 'p' text colour may override the colour inherited from the div